models/public/mozilla-deepspeech-0.6.1/accuracy-check.yml

```diff
@@ -0,0 +1,58 @@
+models:
+  - name: mozilla-deepspeech-0.6.1-with-lm
+    launchers:
+      - framework: dlsdk
+        device: CPU
```

Comment:
please remove device

Reply:
Without `device` accuracy checker (branch `develop`) fails with this message:
```
[...]
  File "/home/ak/progs/_proj/20200123-deepspeech/intel/open_model_zoo/tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py", line 136, in __init__
    dlsdk_launcher_config.validate(self.config, ie_core=self.ie_core)
  File "/home/ak/progs/_proj/20200123-deepspeech/intel/open_model_zoo/tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_config.py", line 96, in validate
    super().validate(entry, field_uri)
  File "/home/ak/progs/_proj/20200123-deepspeech/intel/open_model_zoo/tools/accuracy_checker/accuracy_checker/launcher/launcher.py", line 37, in validate
    super().validate(entry, field_uri)
  File "/home/ak/progs/_proj/20200123-deepspeech/intel/open_model_zoo/tools/accuracy_checker/accuracy_checker/config/config_validator.py", line 117, in validate
    entry, field_uri, "Invalid config for {}: missing required fields: {}".format(field_uri, arguments)
  File "/home/ak/progs/_proj/20200123-deepspeech/intel/open_model_zoo/tools/accuracy_checker/accuracy_checker/config/config_validator.py", line 137, in raise_error
    raise ConfigError(reason)
accuracy_checker.config.config_validator.ConfigError: Invalid config for DLSDK_Launcher: missing required fields: device
```

Reply:
we specify it as command line argument -td <device_name>, it is the reason why we do not use device in config (for reusage the same config for all supported devices)